docs/application/native/guides/compliations/watchface-complication-provider.md

```diff
@@ -0,0 +1,247 @@
+# Watchface complication provider
+
+The watchface complication provider application is a service application to provide complication data.
+Complication provider application provides only data. The watchface application decides how to display the complications.
+The Complication Provider app provides the complication data by using the watchface complication provider api to the watchface app.
+ 
+# Prerequisites
+
+1. To use the Watchface Complication API to communicate with the watchface app, the application has to request permission by adding 
+the following privilege to the tizen-manifest.xml file.
+
+```xml
+<privileges>
+	<privilege>http://tizen.org/privilege/datasharing</privilege>
+</privileges>
+```
+
+2. To use the functions and data types for the Watchface Complication Provider API, include the <watchface-complication-provider.h>.
+
+```cpp
+#include <watchface-complication-provider.h>
+```
+
+# Complication Data Update
+
+When the request occurs to update the complication data, complication provider app can receive the update request by adding 
+watchface_complication_provider_update_requested_cb(). The watchface app can send the update reqeust and the request also occurs after the 
+complication provider app notifies that the complication data is changed.
+

```

Comment:
"When the request occurs to update the complication data, the complication provider application can receive the updated request by adding..."

I think "updated request" is awkward. "update request" means that the requst to update data, but "updated request" means that the request is updated. If you think "update request" is not right, I think it's better to remove "update" word. 

"The watchface application can send the update request and the request occurs only after the complication provider application notifies that that the complication data is changed."

And request can be occured when watchface application wants, not only after the provider application notifies. So there are two cases to occur request, 1. watchface application wants to request. 2. complication provider notifies because the data is updated.

Reply:
@sukhyungkang, thanks for explanation. 
Please can it be rephrased as:
When the request occurs to update the complication data, the complication provider application can receive the request by adding 'watchface_complication_provider_update_requested_cb()'.
The watchface application can send the update request and the request occurs only after the complication provider application notifies that that the complication data is changed.

Reply:
" occurs only after "
not only